openmdao/core/tests/test_approx_derivs.py

```diff
@@ -508,6 +508,46 @@ def compute(self, inputs, outputs):
 
         assert_rel_error(self, derivs['comp.y']['p1.x'], [[4.0]], 1e-6)
 
+    def test_newton_with_densejac_under_full_model_fd(self):
+        # Basic sellar test.
+
+        prob = self.prob = Problem()
```

Comment:
is the assignment to `self.prob` actually needed in this test?

Reply:
only for complex step tests